pkg/otlp/exporterconfig/conditions.go

```diff
@@ -0,0 +1,3 @@
+package exporterconfig
+
+const ConfigConditionType = "OTLPExporterConfigSecret"
```

Comment:
AFAICS, this condition isn't used outside of this package, so i'm wondering if we really need to add yet another condition to the DynaKube